src/scripts/site.js

```diff
@@ -479,21 +479,21 @@ var AppInfo = {};
     function loadPlugins(appHost, browser, shell) {
         console.debug('loading installed plugins');
         var list = [
-            'components/playback/playaccessvalidation',
-            'components/playback/experimentalwarnings',
-            'components/htmlAudioPlayer/plugin',
-            'components/htmlVideoPlayer/plugin',
-            'components/photoPlayer/plugin',
-            'components/youtubeplayer/plugin',
-            'components/backdropScreensaver/plugin',
-            'components/logoScreensaver/plugin'
+            'plugins/playAccessValidation/plugin',
+            'plugins/experimentalWarnings/plugin',
+            'plugins/htmlAudioPlayer/plugin',
+            'plugins/htmlVideoPlayer/plugin',
+            'plugins/photoPlayer/plugin',
+            'plugins/youtubePlayer/plugin',
+            'plugins/backdropScreensaver/plugin',
+            'plugins/logoScreensaver/plugin'
         ];
 
         if (appHost.supports('remotecontrol')) {
-            list.push('components/sessionPlayer');
+            list.push('plugins/sessionPlayer/plugin');
 
             if (browser.chrome || browser.opera) {
```

Comment:
Completely unrelated, but we can also add the new Edge Chromium there I suppose. Do you know if it's under the "Chrome" spectrum or it has its own name?

Reply:
I don't think we detect it specifically. I don't have it, so no idea what it'd show up as. Someone who has Windows available should check.